recipes/mqc/run_test.pl

```diff
@@ -0,0 +1,11 @@
+# Test file for testing Perl modules installation
+$|=1;
```

Comment:
Similar to python tests you can add these for perl as well. E.g. here: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/perl-gd/meta.yaml#L29